apps/newsletters-api/src/app/routes/newsletters.ts

```diff
@@ -12,30 +12,45 @@ import {
 	mapStorageFailureReasonToStatusCode,
 } from '../responses';
 
+interface IListNewslettersQuerystring {
+	includeCancelled?: string;
+}
 export function registerNewsletterRoutes(app: FastifyInstance) {
 	// not using the makeSuccess function on this route as
 	// we are emulating the response of the legacy API
-	app.get('/api/legacy/newsletters', async (req, res) => {
-		const storageResponse = await newsletterStore.list();
-		if (!storageResponse.ok) {
-			return res
-				.status(mapStorageFailureReasonToStatusCode(storageResponse.reason))
-				.send(makeErrorResponse(storageResponse.message));
-		}
+	app.get<{ Querystring: IListNewslettersQuerystring }>(
```

Comment:
The legacy route should behave the same was as the current API endpoint (to avoid migration panic),   so we should not add the `includeCancelled` param on the legacy route but always include the cancelled newsletters by default.

Reply:
The intent here was to facilitate the param passing from the existing api.

It looks like it used to do this but doesn't anymore and I was thrown by the old config in fastly:
![Screenshot 2023-07-05 at 15 04 30](https://github.com/guardian/newsletters-nx/assets/3277259/a18bf654-47fc-432f-8495-4f2aeff21674)

That is only actually set on the code environment and we are not writing a `newsletters-including-cancelled` file to s3. There is one in the dev bucket, so it's eith something that was thought about and never released or partially deleted. I will add a ticket to remove the VCL snipped from Fastly to save confusion in the future. 

The short version of this is, we don't need this change. Closing the PR.

Thanks for looking